### PR TITLE
Use jsonnet to generate prometheus_rule_prow.yaml

### PR DESCRIPTION
**Pull request by hongkailiu**

### CODE REVIEW

File: cluster/ci/monitoring/mixins/Makefile [hunk 1]
```diff
@@ -26,6 +27,10 @@ tide.json:
 	@echo "Generating tide.json ..."
 	jsonnet -J vendor/ grafana_dashboards/tide.jsonnet > ../$@
 
+prometheus_rule_prow.yaml:
+	@echo "Generating prometheus_rule_prow.yaml ..."
+	jsonnet  ./prometheus/prometheus_rule_prow.jsonnet | gojsontoyaml > ../$@
```
Comment (stevekuznetsov) [inline cluster/ci/monitoring/mixins/Makefile]: Do we need to install this in the container image for the presubmit?

Comment (hongkailiu) [reply cluster/ci/monitoring/mixins/Makefile]: You can see that our tide did not complain.
[This tool is added already](https://github.com/openshift/release/blob/master/projects/origin-release/dashboards-validation/Dockerfile#L9) by our friend @paulfantom from monitoring team. ^_^

Comment (stevekuznetsov) [reply cluster/ci/monitoring/mixins/Makefile]: sweet!

**Pull request merged**